src/calculateRentalCost.js

```diff
@@ -3,8 +3,22 @@
  *
  * @return {number}
  */
-function calculateRentalCost(days) {
-  // write code here
+function calculateRentalCost(numberOfDays) {
+  const basePrice = 40;
+  const MIDDLE_TERM = 3;
+  const LONG_TERM = 7;
+  const MIDDLE_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (numberOfDays >= LONG_TERM) {
+    return numberOfDays * basePrice - LONG_TERM_DISCOUNT;
```

Comment:
expression ``numberOfDays * basePrice`` can be used as a variable to avoid duplicating code